### PR TITLE
Db 4053 run whois integration test parallel

### PR DESCRIPTION
**Pull request by MiguelAHM**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -924,20 +928,14 @@
...
-                    <perCoreThreadCount>true</perCoreThreadCount>
+
+                    <!-- parallelize -->
                     <parallel>classes</parallel>
-                    <groups>${test.groups}</groups>
-                    <runOrder>random</runOrder> -->
-                    <!-- works (tests pass) -->
-                   <!-- <groups>${test.groups}</groups>-->
+                    <useUnlimitedThreads>true</useUnlimitedThreads>
+
```
Comment (maggarwal13) [inline pom.xml:935]: Dupliate declaration. We already specified in 903 line number

File: pom.xml [hunk 2]
```diff
@@ -899,8 +899,12 @@
                         <exclude>${test.files.integration}</exclude>
                     </excludes>
                     <runOrder>random</runOrder>
-                    <forkCount>1C</forkCount>
-                    <threadCount>1</threadCount>
+                    <!-- parallelize -->
+                    <parallel>classesAndMethods</parallel>
+                    <useUnlimitedThreads>true</useUnlimitedThreads>
+                    <threadCountClasses>7</threadCountClasses>
+                    <threadCountMethods>3</threadCountMethods>
```
Comment (maggarwal13) [inline pom.xml:906]: How we came up with this number?  If lets say a gitlab runner is not a multi-core does this count means more waiting for method execution?

**Comment by maggarwal13** [comment]
Have we run the tests multiple times to check for flaky tests? On local as well as on gitlab

File: pom.xml [hunk 3]
```diff
@@ -899,8 +899,12 @@
                         <exclude>${test.files.integration}</exclude>
                     </excludes>
                     <runOrder>random</runOrder>
-                    <forkCount>1C</forkCount>
-                    <threadCount>1</threadCount>
+                    <!-- parallelize -->
```
Comment (eshryane) [inline pom.xml:902]: Don't change the config for unit tests, they are already running OK in parallel

**Pull request closed without merging**